### ISSUE TITLE
ModuleNotFoundError: No module named 'run_ocr'

### ISSUE THREAD

**Issue by myrison** [plamola/ocr-joplin-notes#8]
Thanks for all of the recent changes & clean-up you've made to this.  I was testing the latest Docker builds and starting with 0.3.0 and persisting to the latest, they all return the error: No module named 'run_ocr' when issuing the build command below.  My python skills are pretty weak so I'm not able to troubleshoot this one on my own.  

Could you have a look when you have time please?

``$ docker run  --env-file ./docker-env  --network="host" plamola/ocr-joplin-notes:0.3.3 python -m ocr_joplin_notes.cli --mode=FULL_RUN --tag=ocr1``

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.9/site-packages/ocr_joplin_notes/cli.py", line 6, in <module>
    import run_ocr
ModuleNotFoundError: No module named 'run_ocr'
```

**Comment by plamola** [comment]
Sorry about that. I had the imports messed up and forgot to actually test the distribution version.

Should now be fixed in 0.3.4

**Comment by myrison** [comment]
Confirmed, that fixed it.  Thanks for the quick reply and for the entire repo.  Much appreciated.
